Documentation/ApiOverview/Backend/LoginProvider.rst

```diff
@@ -80,17 +80,17 @@ View requirements:
 ..  code-block:: html
     :caption: EXT:my_sitepackage/Resources/Private/Templates/MyLoginForm.html
 
-	<f:layout name="Login" />
-	<f:section name="loginFormFields">
-		<div class="form-group t3js-login-openid-section" id="t3-login-openid_url-section">
-			<div class="input-group">
-				<input type="text" id="openid_url" name="openid_url" value="{presetOpenId}" autofocus="autofocus" placeholder="{f:translate(key: 'openId', extensionName: 'openid')}" class="form-control input-login t3js-clearable t3js-login-openid-field" />
-				<div class="input-group-addon">
-					<span class="fa fa-openid"></span>
-				</div>
-			</div>
-		</div>
-	</f:section>
+    <f:layout name="Login" />
+    <f:section name="loginFormFields">
+        <div class="form-group t3js-login-openid-section" id="t3-login-openid_url-section">
+            <div class="input-group">
+                <input type="text" id="openid_url" name="openid_url" value="{presetOpenId}" autofocus="autofocus" placeholder="{f:translate(key: 'openId', extensionName: 'openid')}" class="form-control input-login t3js-clearable t3js-login-openid-field" />
```

Comment:
While add it, please break this line as long lines in code blocks will cause horizontal srolling